examples/fluids/src/setuplibceed.c

```diff
@@ -549,7 +549,7 @@ PetscErrorCode SetupContextForProblems(Ceed ceed, CeedData ceed_data,
     CeedQFunctionContextCreate(ceed, &ceed_data->dc_context);
     CeedQFunctionContextSetData(ceed_data->dc_context, CEED_MEM_HOST,
                                 CEED_USE_POINTER,
-                                sizeof phys->dc_ctx, phys->dc_ctx);
+                                sizeof(*phys->dc_ctx), phys->dc_ctx);
```

Comment:
@jedbrown Good guess yesterday

Reply:
Was this enough to get it working on GPUs? @LeilaGhaffari What does your simplest GPU failure look like?